.zenodo.json

```
@@ -32,6 +26,12 @@
       "orcid": "0000-0002-0157-030X",
       "type": "Researcher"
     },
+    {
+      "affiliation": "Florey Institute of Neuroscience and Mental Health",
+      "name": "Smith, Robert E.",
+      "orcid": "0000-0003-3636-4642",
+      "type": "Researcher"
+    },
```

Could not tell if there is any order in this file

```suggestion
    },
    {
      "affiliation": "Neuro Data Science ORIGAMI Laboratory, McConnell Brain Imaging Centre, Faculty of Medicine, McGill University, Montréal, Canada",
      "name": "Gau, Rémi",
      "orcid": "0000-0002-1535-9767"
    },    
```

---

The order gets auto-updated by a script, so this is fine.